style.css

```diff
@@ -0,0 +1,117 @@
+.flex-row {
+  display: flex;
+  flex-direction: row;
+}
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.container-header {
+  justify-content: space-between;
+  align-items: center;
+  width: 375px;
+  height: 92px;
+  background: #3c3a39;
+}
+
+.logo {
+  padding-left: 14px;
+  width: 70px;
+  height: 24px;
+  line-height: 133%;
+  align-items: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.menu {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  width: 40px;
+  height: 40px;
+  right: 14px;
+  top: calc(50% - 40px / 2);
+  border-radius: 8px;
+}
+
+.mid-section {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  background: #1c1a19;
+  background-image: url("image/vector-bg.png");
+  background-repeat: no-repeat;
+  background-size: 23vh;
+  background-position: bottom right;
+}
+
+h1 {
+  padding-left: 18px;
+  padding-top: 100px;
+  width: 327px;
+  height: 150px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 56px;
+  line-height: 64px;
+  margin: 0;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
```

Comment:
Kindly update all your text elements to use the same font family used in the figma design🙏

Reply:
If i use the fonts provided by Figma, I get stylelint errors during checks.
<img width="797" alt="Screenshot 2023-05-02 at 5 26 09 PM" src="https://user-images.githubusercontent.com/125091283/235812453-ba7d7d50-0998-477e-972f-450d5f4ef1f1.png">

Reply:
Hi @joshield 

- You need to import those fonts from font providers like google sheets. You can review this [lesson](https://www.freecodecamp.org/news/how-to-use-google-fonts-in-your-next-web-design-project-e1ad48f1adfa/) to learn more. After importing the fonts, it's important to provide a fallback font in case the font fails to load like so `font-family: 'Roboto', sans-serif;`
- Also, the whitespace highlighted before still exists in your project. Please check the styling applied to the section tag, wrapping the social icons and the logo. The hardcoded widths are causing the whitespace.
- I forgot to mention this before, but please change the `section` tag wrapping the social icons to be a `div` tag instead. This will improve the semantics of your page because `section` tags are used to highlight individual sections of a page, such as headline, works, contact etc.